exec/jdbc-all/pom.xml

```diff
@@ -388,51 +396,52 @@
               <include>*:*</include>
             </includes>
             <excludes>
-              <exclude>io.protostuff:*</exclude>
```

Comment:
@cgivre All the diff noise here comes about because I decided to sort this list alphabetically in the hopes that contributors will keep it sorted from here on, making checking what's in it that much easier. I can revert the sorting if preferred though.